CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Changed
 
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.142.12`
```

Comment:
Maybe this should go to "### Fixed"

Reply:
in ###fixed we should only list the bugs that were fixed by the update, not the update itself...

Reply:
I wanted to comment the next line actually, "Display `Config::MdnsEnabled` as true by default." is a fix, but not very important though